config/source/multi/deployments/adapter.yaml

```diff
@@ -68,10 +68,10 @@ spec:
         resources:
           requests:
             cpu: 100m
-            memory: 100Mi
+            memory: 250Mi
           limits:
             cpu: 200m
-            memory: 150Mi
+            memory: 300Mi
 
         ports:
```

Comment:
relevant for the "single" too?

Reply:
the single does not set any requests/limits.